torch/distributed/algorithms/ddp_comm_hooks/optimizer_overlap_hooks.py

```diff
@@ -50,6 +57,7 @@ def _hook_then_optimizer(
     .. warning ::
         This API is experimental adn subject to change.
     """
+    _has_set_params = optimizer_state.params_to_optimize is not None
```

Comment:
Nit: Since this is a local variable, you can name it `has_set_params` (without an underscore).